Add Encointer to Common Good Parachains (#3021)
w3f/polkadot-wiki: docs/learn/learn-common-good-chains.md

Because public utility chains add functionality beyond the scope of the Relay Chain, they will
likely be approved by the network stakeholders only in rare scenarios.

## Common Good Chains

### Statemint

[Statemint](https://github.com/paritytech/cumulus/tree/master/polkadot-parachains/statemint) 
(and its cousin *Statemine* on Kusama) are the first common good parachains.

Statemint is a public utility chain in that it adds functionality not available in the Relay Chain, 
namely, the creation and management of assets. Statemint will support both fungible and non-fungible 
assets. The chain offers an interface similar to ERC-20 for fungible assets and ERC-721 for non-fungible 

> assets. These interfaces are in the logic of the chain itself; by encoding this logic directly into the 

Reviewer: I see that you wanted to keep it uniform and replace tokens with assets. I just ran a quick online search and I see people referring to ERC-20 for fungible tokens and ERC-721 for Non-fungible tokens. 

In the context of Statemint, which is often referred to as an Assets parachain, I guess it is okay to use tokens and assets interchangeably. No action required.

Author: Updated.

Statemint will require the Relay Chain's "root origin", i.e. a referendum. Some of the other logic
(like privileged asset functionality) will defer to the Relay Chain's Council.

### Encointer

Encointer will become Kusama's second common good parachain, which adds logic to the Relay Chain
that aims to bring financial inclusivity to WEB3 and mitigate Sybil attacks with a novel 
Proof of Personhood (PoP) system for unique identity. 

Encointer offers a framework that, in principle, allows for any group of real people to create, 
distribute and use their own digital community tokens.

Encointer aims to invert the [Cantillon Effect](https://www.newworldencyclopedia.org/entry/Richard_Cantillon), 
where money is issued at the bottom, and not as credit to businesses or creditworthy individuals. This way, 
every individual gets a [universal basic income (UBI)](https://book.encointer.org/economics-ubi.html).

To resist Sybil attacks, the Encointer protocol uses a PoP mechanism to foster a 

> unique identity system. The notion is that a person can only be in one place at one time. 

Reviewer: ```suggestion
unique identity system. The notion is that a person can only be present at one place at a given time.
```

Author: Updated.

Participants are requested to attend physical key-signing ceremonies with small groups of random people 
at randomized locations, where these local meetings are part of one global ceremony that co-occur. 
Participants use the Encointer wallet app to participate in these ceremonies, and the wallet enables the 

> management of local community currencies.

Reviewer: ```suggestion
management of local community currencies. Watch an Encointer ceremony in action in [this video](https://www.youtube.com/watch?v=tcgpCCYBqko)
```

Author: Updated.

The protocol involves [other mechanisms](https://book.encointer.org/ssi.html#privacy-considerations) to 
protect the privacy of users in addition to the physical key-signing ceremonies.

> Encointer falls under common good logic as it offers a sybil defense mechanism and a basis for digital 

Reviewer: ```suggestion
Encointer falls under common good logic as it offers a Sybil defense mechanism and a basis for digital 
```

Author: Updated.

> democracy. This can also be adapted by system level chains who can use the unique identity system to 

Reviewer: ```suggestion
democracy. This can also be adapted by system level chains which can use the unique identity system to 
```

Author: Updated.
